lightningd/lightningd.c

```diff
@@ -863,6 +864,7 @@ static struct feature_set *default_features(const tal_t *ctx)
 		OPTIONAL_FEATURE(OPT_ANCHOR_OUTPUTS),
 		OPTIONAL_FEATURE(OPT_QUIESCE),
 		OPTIONAL_FEATURE(OPT_ONION_MESSAGES),
+		OPTIONAL_FEATURE(OPT_SPLICE),
```

Comment:
You'll notice that EXPERIMENTAL_FEATURES is gone in the rebase.  You need to set this with an explicit `--experimental-splicing` option...